ros/src/computing/perception/localization/packages/orb_localizer/nodes/place_recognizer/image_publisher.cpp

```diff
@@ -0,0 +1,62 @@
+/*
+ * image_publisher.cpp
+ *
+ *  Created on: May 21, 2017
+ *      Author: sujiwo
+ */
+
+/*
+ * The purpose of this node is to publish a single image. It is used to test
+ * the performance of place recognizer.
+ */
+
+
+#include <iostream>
+#include <string>
+#include <ros/ros.h>
+
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <unistd.h>
+
+#include <opencv2/highgui/highgui.hpp>
+#include <sensor_msgs/Image.h>
+#include <sensor_msgs/image_encodings.h>
+#include <cv_bridge/cv_bridge.h>
+#include <image_transport/image_transport.h>
+
+
+using namespace std;
```

Comment:
ROS Style guide forbids this: https://google.github.io/styleguide/cppguide.html#Namespaces